src/v3/src/types/schema.ts

```diff
@@ -454,11 +454,17 @@ export interface AccordionPanelElement extends UISchemaElement {
   };
 }
 
+export type ImageName = 'app_icon'
+| 'device_icon'
+| 'mobile_device_icon'
+| 'location_icon'
+| 'phone_icon';
+
 export interface ImageWithTextElement extends UISchemaElement {
   type: 'ImageWithText';
   options: {
     id: string;
-    SVGIcon: FunctionComponent;
+    SVGIcon: FunctionComponent | ImageName;
```

Comment:
Do we need to use a string here for these images or could we also directly import the component and pass it as an option?

Reply:
Yep youre absolutely right.  We can keep it as FunctionComponent type and just import the Image components directly in the transformers and pass them in.  No need for the string type and unnecessary util function